controller/persistence/__init__.py

```diff
@@ -0,0 +1,164 @@
+from threading import Lock
+import os
+import os.path
+from persistence.mongo_client import Database
+
+
+class DataStorage:
+    """Centralized Access to File System and Database"""
```

Comment:
Format comments in comment like comments in ControllerManager

Reply:
You use a different comment style compared to the rest of the project. We would like to keep it uniformed:
First block function description
Second block input parameter description (optional, only if parameter available)
third block output parameter description (optional, only if output returned)

example:
        """
        Get the generated model zip for one AutoML 
        ---
        Parameter
        1. session id
        ---
        Return a .zip containing the model and executable script
        """

Reply:
I was hoping this would not be required to be honest.
Usually extensive docstrings don't add much value and end up being noise in the code (eg. what could session_id be other than the id of a session?).

The docstrings in this class have (mostly) copy-and-pastable snippets which are syntax highlighted in the editor:
![image](https://user-images.githubusercontent.com/33757567/173671166-d3ca2fe9-aaca-4287-bb28-c49f19a073d9.png)

I'll extend the docstrings, but I'll come back to the topic at some point ;)